apps/dashboard/src/components/workflow-editor/steps/email/email-editor.tsx

```diff
@@ -14,13 +13,10 @@ export const EmailEditor = (props: EmailEditorProps) => {
 
   return (
     <>
-      <EmailPreviewHeader className="p-3" />
-      <EmailTabsEditSection>{subject && getComponentByType({ component: subject.component })}</EmailTabsEditSection>
-
+      <EmailPreviewHeader />
```

Comment:
I think removing the `p-3` here made the header appear in different places for the tabs.

Reply:
I'm fixing that now.